.gitattributes

```diff
@@ -0,0 +1,8 @@
+###############################################################################
+# Set default behavior to automatically normalize line endings.
+###############################################################################
+* text=auto
+
+# Ensure shell scripts use LF line endings (linux only accepts LF)
+*.sh eol=lf
+*.ps1 eol=lf
```

Comment:
This is unnecessary:
* we don't use any PowerShell scripts;
* [PowerShell doesn't care about line endings](https://stackoverflow.com/a/73981562/393146).

```suggestion
```

Reply:
I'll remove it on the basis that you have no .ps1 scripts.
But FYI, while _powershell_ doesn't care about line endings, _linux_ does. And if you start your ps1 script with `#!/env/pwsh` or something similar, that file had better use LF line endings or linux will be unable to find `pwsh` due to the extra CR at the end of that line. That's why I added this line to all my `.gitattributes` files.